Truck.js

```diff
@@ -1,6 +1,39 @@
 var Car = require('./Car.js');
 
+
+
 //Implement your Truck here
-function Truck() {};
+function Truck(make, model, year, color, seats, passengers, capacity) {
+	var cargo = 0
```

Comment:
Here, you're going to want ```this.cargo = cargo``` and then add ```cargo``` as a parameter in your Truck constructor.